pages/common/initdb.md

```diff
@@ -0,0 +1,8 @@
+# initdb
+
+> Create a PostgreSQL database on disk.
+> More information: <https://www.postgresql.org/docs/9.5/app-initdb.html>.
+
+- Create a database at /usr/local/var/postgres:
+
+`initdb -D /usr/local/var/postgres`
```

Comment:
Please wrap the location within backticks (``)

Reply:
Shouldn't that be `{{token_syntax}}`, @agnivade?

Reply:
My bad. Yes it should be the token syntax

Reply:
```suggestion
`initdb -D {{/usr/local/var/postgres}}`
```

(also, is there a long form of that flag?